clusters/app.ci/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -28,3 +28,57 @@ items:
   - apiGroup: rbac.authorization.k8s.io
     kind: Group
     name: ci-admins
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: ci-admins-cluster-reader
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-reader
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: ci-admins
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: ci-admins-aggregated
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: ci-admins-aggregated
+  subjects:
+  - apiGroup: rbac.authorization.k8s.io
+    kind: Group
+    name: ci-admins
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    name: ci-admins-aggregated
+  aggregationRule:
+    clusterRoleSelectors:
+    - matchLabels:
+        dptp.openshift.io/ci-admins-aggregated: "true"
+  rules: []
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: ClusterRole
+  metadata:
+    annotations:
+      openshift.io/description: This cluster role extends the build-in cluster-reader
```

Comment:
it doesn't extend it, it replaces it or not?

Reply:
Done.